workspaces-tests/src/bin/native_multisig.rs

```diff
@@ -0,0 +1,110 @@
+// Ignore
+pub fn main() {}
+
+use std::fmt::Display;
+
+use near_contract_tools::{
+    approval::{
+        native_transaction_action::{self, NativeTransactionAction},
+        simple_multisig::{AccountApprover, ApprovalState, Configuration},
+        ApprovalManager,
+    },
+    rbac::Rbac,
+    slot::Slot,
+    Rbac,
+};
```

Comment:
[Non-blocking] [Suggestion] Important to have clear documentation what this workspace smart contract is (simple multisig + native transactions) as opposed to a different multisig implementation than other workspace-tests simple_multisig.